admission-plugin.yaml

```diff
@@ -0,0 +1,7 @@
+admissionConfig:
+  pluginConfig:
+    openshift.io/ImagePolicy:
+      configuration:
+        kind: ImagePolicyConfig
+        apiVersion: v1
+        qualifyRulesFilename: /home/centos/rules
```

Comment:
since this admission plugin is distinct, we can just inline the rules in the ImagePolicyConfig struct. if we want this externalized, we'd reference the whole plugin config externally:

master-config.yaml
```yaml
...
admissionConfig:
  pluginConfig:
    openshift.io/ImagePolicy:
      location: /path/to/plugin.yaml
...
```

plugin.yaml:
```yaml
kind: ImagePolicyConfig
apiVersion: v1
rules:
- ...
- ...
- ...
```